docs/contributing.md

```diff
@@ -78,14 +78,14 @@ Ready to contribute? Here's how to set up _geemap_ for local development.
 5. When you're done making changes, check that your changes pass flake8 and the tests, including testing other Python versions with tox:
```

Comment:
Not for the commit, but setting up https://github.com/pre-commit/pre-commit or simillar would be a good idea to automate this kind of cleanup

Reply:
Ack. Good idea!